mitosheet/mitosheet/public/v3/sheet_functions/string_functions.py

```diff
@@ -8,18 +8,79 @@
 
 NOTE: This file is alphabetical order!
 """
-from typing import Union
+from typing import Optional, Union
 import sys
 import numpy as np
 
 import pandas as pd
 
+from mitosheet.public.v3.errors import handle_sheet_function_errors
 from mitosheet.public.v3.sheet_functions.utils import get_final_result_series_or_primitive
-from mitosheet.public.v3.types.decorators import cast_values_in_arg_to_type
-from mitosheet.public.v3.types.sheet_function_types import StringFunctionReturnType
+from mitosheet.public.v3.types.decorators import cast_values_in_args_to_type, cast_values_in_arg_to_type
+from mitosheet.public.v3.types.sheet_function_types import StringInputType, StringRestrictedInputType, StringFunctionReturnType, IntRestrictedInputType
 
-@cast_values_in_arg_to_type('str')
-def CONCAT(*argv: pd.Series) -> StringFunctionReturnType:
+
+@cast_values_in_arg_to_type('string', 'str')
+@cast_values_in_arg_to_type('num_chars', 'int')
+@handle_sheet_function_errors
+def LEFT(string: StringRestrictedInputType, num_chars: IntRestrictedInputType=None) -> StringFunctionReturnType:
```

Comment:
On our path to handling these functions more like Excel, let's change the parameters to match Excel. This touches the docs, so we can leave it till later again. 

<img width="724" alt="Screenshot 2023-03-18 at 11 25 06 AM" src="https://user-images.githubusercontent.com/18709905/226115005-3e993181-3829-47f6-82e1-c25cdf9ecc71.png">

Reply:
We can rename all of these when we do a docs pass.